src/unity/python/turicreate/toolkits/_evaluate_utils.py

```diff
@@ -0,0 +1,85 @@
+def entropy(probs):
+    return _reduce(lambda x, y: x + (y*math.log(1/y, 2) if y > 0 else 0) , probs, 0) / math.log(len(probs),2)
```

Comment:
Please file an issue to move this code to C++ eventually.

Reply:
Filed Issue: #1962